pytorch_lightning/core/lightning.py

```diff
@@ -1917,7 +1917,7 @@ def model_size(self) -> float:
             )
         return get_model_size_mb(self)
 
-    def add_to_queue(self, queue: torch.multiprocessing.SimpleQueue) -> None:
+    def add_to_queue(self, queue: pl.plugins.training_type.ddp_spawn._ExtraQueue) -> None:
```

Comment:
Curious why do we have this in lightningModule? Can we call directly from DDPSpawnPlugin?

Reply:
nvm!